packages/components/src/components/hds/code-block/index.ts

```diff
@@ -140,6 +140,7 @@ export default class HdsCodeBlock extends Component<HdsCodeBlockSignature> {
     const grammar = language ? Prism.languages[language] : undefined;
 
     if (code) {
+      // eslint-disable-next-line ember/no-runloop
```

Comment:
question: should we aim to migrate these to `ember-lifeline` or another recommended solution?

Reply:
Yeah - we should definitely migrate off `ember-runloop` and `ember-render-modifiers`. I can add tickets to the backlog since those changes could be more involved...

Reply:
> Yeah - we should definitely migrate off `ember-runloop` and `ember-render-modifiers`. I can add tickets to the backlog since those changes could be more involved...

@shleewhite @zamoore @alex-ju my 2 cents: before going all-in with such change, I would recommend doing a thorough analysis of the impact/effort (with associated RFC): it may not be as easy/straightforward as it may seem (from what I remember) plus it may require breaking changes in the components' APIs.

Reply:
100% agree

Reply:
yes, to be clear, I was not suggesting we make these changes in this PR.

I find it paradoxical that the ember community suggests these are not recommended approaches when there are few alternatives (if any) out there

Reply:
Tickets: 
- https://hashicorp.atlassian.net/browse/HDS-3884
- https://hashicorp.atlassian.net/browse/HDS-3885